nebula_hw_interfaces/src/nebula_velodyne_hw_interfaces/velodyne_hw_interface.cpp

```diff
@@ -167,13 +167,44 @@ VelodyneStatus VelodyneHwInterface::check_and_set_config(
               << std::endl;
   }
 
-  target_key = "config.fov.start";
-  auto current_cloud_min_angle = tree.get<std::uint16_t>(target_key);
   int setting_cloud_min_angle = sensor_configuration->cloud_min_angle;
-  // Velodyne only allows a maximum of 359 in the setting
-  if (setting_cloud_min_angle == 360) {
-    setting_cloud_min_angle = 359;
+  int setting_cloud_max_angle = sensor_configuration->cloud_max_angle;
+
+  // FIXME: VLP16 has problems for timestamp. Whatch github issue #
+  if (sensor_configuration->sensor_model == SensorModel::VELODYNE_VLP16) {
+    int angle_diff = (setting_cloud_max_angle - setting_cloud_min_angle + 360) % 360;
```

Comment:
Please define the angle we oversize by as a `const int32_t fov_tolerance_deg = 5;` and replace all `5`s below by that constant.